python/pacemaker/_cts/audits.py

```diff
@@ -263,6 +263,23 @@ def __init__(self, cm):
         self.known = []
         self.name = "FileAudit"
 
+    def _find_core_with_coredumpctl(self, node):
+        """Use coredumpctl to find core dumps on the given node."""
+        found = False
+        (_, lsout) = self._cm.rsh(node, "coredumpctl --no-legend")
```

Comment:
This function is almost identical to `_find_core_on_fs()`. The only differences are the `rsh` line and the `log` line. Could call into another helper if we wanted. These are so short that I have no preference.